bl-kernel/admin/views/content.php

```diff
@@ -279,15 +280,16 @@ function table($type) {
 			'type': 'hidden',
 			'name': 'type',
 			'value': 'delete'
-		}))));
+		}).append(jQuery('<input>', {
+			'type': 'hidden',
+			'name': 'hash',
```

Comment:
The variable need to be `tabhash` or `tabHash` for clarify from what is the hash.

Reply:
Yep, makes sense! Will change.